src/crawlee/base_storage_client/_models.py

```diff
@@ -230,3 +231,27 @@ class BatchRequestsOperationResponse(BaseModel):
 
     processed_requests: Annotated[list[ProcessedRequest], Field(alias='processedRequests')]
     unprocessed_requests: Annotated[list[UnprocessedRequest], Field(alias='unprocessedRequests')]
+
+
+@docs_group('Data structures')
+class InternalRequest(BaseModel):
+    """Represents an request in queue client."""
+
+    id: str
+
+    unique_key: str
+
+    order_no: Decimal | None = None
+
+    handled_at: datetime | None
+
+    request: Request
```

Comment:
If this could have an alias to `json_`, we wouldn't break compatibility with storage files from older versions, which is not a big deal, but still it would be nice.

Reply:
Is it unrealistic to add that (de)serialization logic though? 

If it can't be done, we would need to detect the case when a user tries to load an outdated memory storage file and throw an exception, which might be harder than ensuring compatibility.

Reply:
Makes sense. Updated